channeldb/graph.go

```diff
@@ -3201,6 +3205,9 @@ func putChanEdgePolicy(edges, nodes *bbolt.Bucket, edge *ChannelEdgePolicy,
 	if err := binary.Write(&b, byteOrder, uint64(edge.FeeProportionalMillionths)); err != nil {
 		return err
 	}
+	if err := binary.Write(&b, byteOrder, uint64(edge.MaxHTLC)); err != nil {
```

Comment:
must be moved right before opaque data field. Also should only be written if flag is set?

Reply:
fixed